### PR TITLE
Add modular files

### PR DESCRIPTION
**Pull request by ThatLing**
Please tell me if I missed something

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,70 +1,29 @@
...
+/code/controllers/master.dm @ThatLing
+/code/controllers/failsafe.dm @ThatLing
+/code/controllers/subsystem.dm @ThatLing
+/code/controllers/subsystem/ @ThatLing
+
+
+#Nichlas0010
+
+/code/modules/shuttle/ @Nichlas0010
+code/datums/shuttles.dm @Nichlas0010
```
Comment (Amelia0010) [inline .github/CODEOWNERS]: /code/datums...

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -391,7 +379,7 @@ There is no strict process when it comes to merging pull requests. Pull requests
...
 
-Regarding sprites & sounds, you must credit the artist and possibly the codebase. All /tg/station assets including icons and sound are under a [Creative Commons 3.0 BY-SA license](https://creativecommons.org/licenses/by-sa/3.0/) unless otherwise indicated. However if you are porting assets from GoonStation or usually any assets under the [Creative Commons 3.0 BY-NC-SA license](https://creativecommons.org/licenses/by-nc-sa/3.0/) are to go into the 'goon' folder of the /tg/station codebase.
+Regarding sprites & sounds, you must credit the artist and possibly the codebase. All yogstation assets including icons and sound are under a [Creative Commons 3.0 BY-SA license](https://creativecommons.org/licenses/by-sa/3.0/) unless otherwise indicated. However if you are porting assets from GoonStation or usually any assets under the [Creative Commons 3.0 BY-NC-SA license](https://creativecommons.org/licenses/by-nc-sa/3.0/) are to go into the 'goon' folder of the yogstation codebase.
 
 ## Banned content
 Do not add any of the following in a Pull Request or risk getting the PR closed:
```
Comment (Amelia0010) [inline .github/CONTRIBUTING.md:385]: get rid of the

> * Code adding, removing, or updating the availability of alien races/species/human mutants without prior approval. Pull requests attempting to add or remove features from said races/species/mutants require prior approval as well.

As well. We don't have the policy.

File: yogstation/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
+###### Special thanks to [HippieStation](https://github.com/HippieStation/HippieStation/blob/master/hippiestation/README.md) for the help.
+
+# Introduction to the "yogstation" Folder
+
+## Why do we use it?
+
+To keep up-to-date with /tg/station while trying to avoid merge conflicts as much as possible, we modularize everything. This is the same method [HippieStation](https://github.com/yogsStation/yogsStation/tree/master/yogsstation) uses. The more code in the "yogstation" folder, the fewer conflicts will exist in future updates.
```
Comment (Amelia0010) [inline yogstation/README.md]: >This is the same method \[HippieStation\]\(https://github.com/yogsStation/yogsStation/tree/master/yogsstation\) uses.

Comment (nichlas0010) [inline yogstation/README.md]: this link should go to hippie

**Review by Amelia0010 — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -6,38 +6,26 @@ See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and fo
...
+We have a [list of guides on the wiki](https://wiki.yogstation.net/wiki/Guides#Development_and_Contribution_Guides) that will help you get started contributing to yogstation with Git and Dream Maker. For beginners, it is recommended you work on small projects like bugfixes at first. If you need help learning to program in BYOND, check out this [repository of resources](http://www.byond.com/developer/articles/resources).
 
-There is an open list of approachable issues for [your inspiration here](https://github.com/tgstation/-tg-station/issues?q=is%3Aopen+is%3Aissue+label%3A%22Good+First+Issue%22).
-
-You can of course, as always, ask for help at [#coderbus](irc://irc.rizon.net/coderbus) on irc.rizon.net. We're just here to have fun and help out, so please don't expect professional support.
+You can of course, as always, ask for help in #coder-public on the [discord](https://discord.gg/0keg6hQH05Ha8OfO). We're just here to have fun and help out, so please don't expect professional support.
 
 ## Meet the Team
```
Comment (simonset55) [inline .github/CONTRIBUTING.md:23]: I don't understand this, we have a team?

File: .github/CONTRIBUTING.md [hunk 5]
```diff
@@ -6,38 +6,26 @@ See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and fo
...
 
-**Design Lead**
-
-The Design Lead has the final say on what gameplay changes get into and out of the game. He or she has full veto power on any feature or balance additions, changes, or removals, and establishes a general, personally-preferred direction for the game.
-
 **Headcoder**
```
Comment (simonset55) [inline .github/CONTRIBUTING.md:25]: He's inactive lol

File: .github/CONTRIBUTING.md [hunk 6]
```diff
@@ -6,38 +6,26 @@ See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and fo
...
 
 The Headcoder is responsible for controlling, adding, and removing maintainers from the project. In addition to filling the role of a normal maintainer, they have sole authority on who becomes a maintainer, as well as who remains a maintainer and who does not.
 
-**Art Director**
-
-The Art Director controls sprites and aesthetic that get into the game. While sprites for brand-new additions are generally accepted regardless of quality, modified current art assets fall to the Art Director, who can decide whether or not a sprite tweak is both merited and a suitable replacement.
-
-They also control the general "perspective" of the game - how sprites should generally look, especially the angle from which they're viewed. An example of this is the [3/4 perspective](http://static.tvtropes.org/pmwiki/pub/images/kakarikovillage.gif), which is a bird's eye view from above the object being viewed.
-
 **Maintainers**
```
Comment (simonset55) [inline .github/CONTRIBUTING.md:29]: "quality"

File: .github/CONTRIBUTING.md [hunk 7]
```diff
@@ -139,7 +127,7 @@ Hacky code, such as adding specific checks, is highly discouraged and only allow
 You can avoid hacky code by using object-oriented methodologies, such as overriding a function (called "procs" in DM) or sectioning code into functions and then overriding them as required.
 
 ### No duplicated code
```
Comment (simonset55) [inline .github/CONTRIBUTING.md:129]: What, isn't this what we're all about lmao xdddddd

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (simonset55) [inline .github/CONTRIBUTING.md:385]: Add ERP mechanics

File: .github/CONTRIBUTING.md [hunk 8]
```diff
@@ -6,38 +6,26 @@ See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and fo
 
 ## Introduction
```
Comment (simonset55) [inline .github/CONTRIBUTING.md:7]: This whole thing is very hard for me to understand, please make it more illiterate-friendly

File: .github/CONTRIBUTING.md [hunk 9]
```diff
@@ -391,7 +379,7 @@ There is no strict process when it comes to merging pull requests. Pull requests
 
 If you are porting features/tools from other codebases, you must give them credit where it's due. Typically, crediting them in your pull request and the changelog is the recommended way of doing it. Take note of what license they use though, porting stuff from AGPLv3 and GPLv3 codebases are allowed.
```
Comment (simonset55) [inline .github/CONTRIBUTING.md:380]: From my brain ofc XD

File: .github/CONTRIBUTING.md [hunk 10]
```diff
@@ -6,38 +6,26 @@ See [this page](http://tgstation13.org/wiki/Reporting_Issues) for a guide and fo
...
 
 First things first, we want to make it clear how you can contribute (if you've never contributed before), as well as the kinds of powers the team has over your additions, to avoid any unpleasant surprises if your pull request is closed for a reason you didn't foresee.
 
 ## Getting Started
 
-/tg/station doesn't have a list of goals and features to add; we instead allow freedom for contributors to suggest and create their ideas for the game. That doesn't mean we aren't determined to squash bugs, which unfortunately pop up a lot due to the deep complexity of the game. Here are some useful starting guides, if you want to contribute or if you want to know what challenges you can tackle with zero knowledge about the game's code structure.
+yogstation doesn't have a list of goals and features to add; we instead allow freedom for contributors to suggest and create their ideas for the game. That doesn't mean we aren't determined to squash bugs, which unfortunately pop up a lot due to the deep complexity of the game. Here are some useful starting guides, if you want to contribute or if you want to know what challenges you can tackle with zero knowledge about the game's code structure.
```
Comment (simonset55) [inline .github/CONTRIBUTING.md:15]: Our goal is to resist change at any moment

File: .github/CODEOWNERS [hunk 11]
```diff
@@ -1,70 +1,29 @@
...
+# In the event that multiple org members are to be informed of changes 
+# to the same file or dir, add them to the end under Multiple Owners 
+ 
+# Example
+ 
+#/code/world.dm @GithubNameHere
+#/maps/ @GithubNameHere
+
+
+#ThatLing
```
Comment (simonset55) [inline .github/CODEOWNERS:13]: Who dis nerd

File: yogstation/README.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+  - Should generally be about the reason the change was made, what it was before, or what the change is
+  - Multi-line messages should start with `// yogs start` and use `/* Multi line message here */` for the message itself
+- Single line changes should have `// yogs` or `// yogs - reason`
+
+If you need to mirror a file, or function into a yog-specific file, please leave behind a comment stating where it went.
+
+```
+// yogs start - Mirrored this function in <file> for <reason>
+bunch of shitcode here
+// yogs stop
```
Comment (simonset55) [inline yogstation/README.md:28]: Yes please stop

**Review by simonset55 — COMMENTED** [review]
What the fuck am i doing

File: yogstation/README.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+New actions and spells should use the "yogstation/icons/mob/actions.dmi" file. If it is a spell, put the code for the spell in "yogstation/code/modules/spells." To make sure that the spell uses the Yogs icon, please add "action_icon = 'yogstation/icons/mob/actions.dmi'" and the "action_icon_state" var.
+
+### Reagents
+
+New reagents should go inside "yogstation/code/modules/reagents/drug_reagents.dm." In this case, "drug_reagents" is an example, so please use or create a "toxins.dm" if you are adding a new toxin. Recipes should go inside "yogstation/code/modules/reagents/recipes/drug_reagents.dm." Once again, "drug_reagents" has been used as an example.
+
+---
+
+### Thank you for following these rules! Please contact a maintainer if you have any questions about modular code or the "haven" folder.
```
Comment (Partheo) [inline yogstation/README.md]: "haven" folder- change this to Yogs

**Review by Partheo — CHANGES_REQUESTED** [review]
Looks good otherwise

File: yogstation/README.md (hunk 13 above)
Comment (ThatLing) [reply yogstation/README.md]: Oops, yeah I copied that line from OH.

**Review by Amelia0010 — APPROVED** [review]

**Pull request merged**